rust-src/concordium_base/src/web3id/mod.rs

```diff
@@ -1059,7 +1059,12 @@ impl CredentialSchema {
     }
 }
```

Comment:
Seems like `CredentialSchema` is not used anywhere now, at least not in this file. Maybe it's imported somewhere else, but `attribute_tag` seems irrelevant since tags are not indices now.

Reply:
Forgot to delete this. Will remove.